### ISSUE TITLE
Often mobile themes aren't loading

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#3329]
Fairly regularly, when I open a mobile app in the page editor, the themes aren't loading properly. On Windows/Chrome, I'm pretty sure I've seen this on other browsers, too. I can't see to get the themes to show in the page editor (preview is ok). Here is a snapshot that shows what I'm seeing:

![Mobile themes not loading](http://i.imgur.com/ToMbp.png)

Using build 57d4840af5aa599... on the staging server, which should have the most recent changes for theme loading.

I'm seeing this way too often for us to ship as is.

I seem to be able to reproduce this by:
- Win/Chrome
- Clone custom to create custom2
- Create same content as in our mobile tutorial
- Switch to custom2
- Save, leave editor open
- Browser refresh

At the time I did a browser refresh, I had 5 editors open in total, including the custom2.theme theme editor.

**Comment by billreed63** [comment]
This is the timing issue we have had with onload since @peller turned on async. O have moved around the code tring to find a spot the works best but have had no luck.. @peller was nver able to reproduce this issue..

**Comment by billreed63** [comment]
I can comment the code out of context.js so that deviceTheme.loadDeviceTheme only gets called when the object is created by the requires.  I still see that the css files are added to the head but the style is not applied to the page. If I try to use the debugger the style gets applied.

**Comment by billreed63** [comment]
This also seems to happen more often when the page is open and the browser is refreshed.. If you close the file and reopen inside maqetta the style is applied in most cases..

**Comment by jhpedemonte** [comment]
@peller, please help investigate.

**Comment by billreed63** [comment]
The fix above is temp solutions, we need more robust solutions for M8

**Comment by billreed63** [comment]
Article on firing events when style sheet is loaded http://www.phpied.com/when-is-a-stylesheet-really-loaded/

**Comment by wocmultimedia** [comment]
I think this other strange behaviour is connected to previous one but it's up to you understand it.
I created a new project populated with 3 views and a scrolling_view
I add a picture to the Menu_Item background and I load a .png image for the same button linking to a View using moveTo:... I downloaded the workspace complete. Then I tested the files and result is on this shot were is the picture shown

http://www.wocmultimedia.com/blogstore/maqetta/maq7PictureShown1.jpg

I turned off the PC and restart later but opening the project the picture is not shown anymore eithier in Designer view or Preview mode. Both pictures are still there on the images folder as before. See shot below.

http://www.wocmultimedia.com/blogstore/maqetta/maq7PictureNoshown.jpg

Thanks for helping
